.github/workflows/ci-pre-commit.yml

```
@@ -0,0 +1,15 @@
+name: Format
```

```suggestion
name: Pre-Commit
```

Or is there a reason you didn't change it?